File: ajovanovic9.c
// Angela Jovanovic
// Lab 9c

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void)
{
	srand(time(NULL));

	int num = rand() % 21;
	int i, guess;

	while(guess < 0 || guess > 20)
	{
		printf("Enter a number between 0 and 20: ");
		scanf("%d", &guess);
		if(guess < 0 || guess > 20)
		{
			puts("You entered a number out of range!");
		}
	}
	
	if(guess == num)
	{
		for(i = 0; i < guess; i++)
		{
			printf("You won! You guessed the correct number!\n");
		}
	}
	else if(guess < num)
	{
		printf("You lost...Your guess was lower than my number %d.\n", num);
	}
	else
	{
		printf("You lost...Your guess was higher than my number %d.\n", num);
	}

	return EXIT_SUCCESS;
}

File: ajovanovic9.java
// Angela Jovanovic
// Lab 9c

import java.util.Scanner;
import java.util.Random;

public class ajovanovic9
{
	public static void main(String[] args)
	{
		Random rand = new Random();

		int i;
		int guess = -1;
		int num = rand.nextInt(21);
		Scanner input = new Scanner(System.in);

		while(guess < 0 || guess > 20)
		{
			System.out.println("Enter a number between 0 and 20: ");
			guess = input.nextInt();

			if(guess < 0 || guess > 20)
			{
				System.out.println("You entered a number out of range!");
			}
		}

		if(guess == num)
		{
			for(i = 0; i < guess; i++)
			{
				System.out.println("You win! You guessed the correct number!");
			}
		}
		else if(guess < num)
		{
			System.out.println("You lose...Your guess was lower than my number " + num + ".");
		}
		else{
			System.out.println("You lose...Your guess was higher than my number " + num + ".");
		}
	}
}
